antsibull/build_ansible_commands.py

```diff
@@ -263,8 +262,8 @@ def build_single_impl(dependency_data: DependencyFileData, add_release: bool = T
         make_dist(package_dir, app_ctx.extra['dest_dir'])
 
     # Write changelog and porting guide also to destination directory
-    release_notes.write_changelog_to(deps_dir)
-    release_notes.write_porting_guide_to(deps_dir)
+    release_notes.write_changelog_to(app_ctx.extra['build_data_dir'])
+    release_notes.write_porting_guide_to(app_ctx.extra['build_data_dir'])
```

Comment:
I think we want writing to always use dest_dir and reading to always use build_data_dir.  If those are the same, then it's up to the user to set them to the same directory using command line args.

Reply:
For changelog and porting guide I agree, but for new `.deps` files (and the updated `changelog.yml`, which right now is updated in place - i.e. both read and written from/to the same place) I disagree. These belong in the `build_data_dir`, and not in the directory where tarball is stored.

Reply:
Hmm, IMO the .rst changelog also belongs in build_data_dir since that's where it lives.